libs/platform/button/button.component.html

```diff
@@ -2,7 +2,6 @@
     #fdButton
     fd-button
     [attr.id]="id"
-    [attr.title]="title"
```

Comment:
that's not a proper solution to the problem. When we have a text on the button we shouldn't have a title as the SR will read the text. But when we have a button that only has an icon, we need title.

<img width="300" alt="Screenshot 2024-07-23 at 4 54 41 PM" src="https://github.com/user-attachments/assets/456200da-dedb-485f-b096-9c769ddf8055">

Since we have a `glyph` prop on the button you can add the title attribute conditionally and that would solve the problem